<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>triggerDelayWindow</class>
 <widget class="QWidget" name="triggerDelayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CamSpinBox" name="spinPreFrames">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOK">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="TriggerSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>771</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CamDoubleSpinBox" name="spinPreSeconds">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Frames</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdHundredPercent">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdFiftyPercent">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdZeroPercent">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="CamSpinBox" name="spinPostFrames">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_10">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Post-trigger</string>
   </property>
  </widget>
  <widget class="CamDoubleSpinBox" name="spinPostSeconds">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>7200.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_11">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Frames</string>
   </property>
  </widget>
  <widget class="CamSpinBox" name="spinPreRecFrames">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
  </widget>
  <widget class="CamDoubleSpinBox" name="spinPreRecSeconds">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>7200.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Frames</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_14">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_15">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="text">
    <string>(Time from trigger to end of recording)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="text">
    <string>(Time from trigger to start of recording)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pre-record delay</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_18">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="text">
    <string>(Time from start of recording to trigger)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_19">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pre-trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRes_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time  ▶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdMorePreRecTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>More ◀
Pre-rec Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdResetToDefaults">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>370</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reset To
Defaults</string>
   </property>
  </widget>
  <zorder>cmdCancel</zorder>
  <zorder>cmdOK</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>lblRes_8</zorder>
  <zorder>lblRes_9</zorder>
  <zorder>cmdHundredPercent</zorder>
  <zorder>cmdFiftyPercent</zorder>
  <zorder>cmdZeroPercent</zorder>
  <zorder>lblRes_10</zorder>
  <zorder>lblRes_11</zorder>
  <zorder>lblRes_12</zorder>
  <zorder>spinPreRecSeconds</zorder>
  <zorder>lblRes_13</zorder>
  <zorder>lblRes_14</zorder>
  <zorder>lblRes_15</zorder>
  <zorder>lblRes_16</zorder>
  <zorder>lblRes_17</zorder>
  <zorder>lblRes_18</zorder>
  <zorder>lblRes_19</zorder>
  <zorder>lblRes_20</zorder>
  <zorder>cmdMorePreRecTime</zorder>
  <zorder>cmdResetToDefaults</zorder>
  <zorder>spinPreSeconds</zorder>
  <zorder>spinPostSeconds</zorder>
  <zorder>spinPreRecFrames</zorder>
  <zorder>spinPreFrames</zorder>
  <zorder>spinPostFrames</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CamSpinBox</class>
   <extends>QSpinBox</extends>
   <header>camspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>CamDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>camdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>TriggerSlider</class>
   <extends>QSlider</extends>
   <header>triggerslider.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmdCancel</tabstop>
  <tabstop>cmdOK</tabstop>
  <tabstop>cmdResetToDefaults</tabstop>
  <tabstop>cmdMorePreRecTime</tabstop>
  <tabstop>spinPostFrames</tabstop>
  <tabstop>spinPreFrames</tabstop>
  <tabstop>spinPreRecFrames</tabstop>
  <tabstop>spinPostSeconds</tabstop>
  <tabstop>spinPreSeconds</tabstop>
  <tabstop>spinPreRecSeconds</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>cmdHundredPercent</tabstop>
  <tabstop>cmdFiftyPercent</tabstop>
  <tabstop>cmdZeroPercent</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
